spec/order_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'order'
+
+describe Order do
+  let(:small_order){1}
+  let(:large_order){5}
+  let(:messager){double(:messager, send_sms: true)}
+  let(:messager_klass){double(:messager_klass, :new => messager)}
+  let(:menu_item){double(:menu_item, name: 'chicken satay', :price => 4.50)}
+  subject(:order){described_class.new(messager_klass)}
+
+  describe '#initialize' do
+
+    it 'instantiates a new messager' do
+      expect(order.messager_klass).to eq(messager)
+    end
+
+  end
+
+ describe '#add_item' do
+
+   it 'adds an item and quantity to the order history' do
+     order.add_item(menu_item, small_order)
+     expect(order.history).to include(menu_item => small_order)
+   end
+
+   it 'returns a string confirming the item added and the quantity' do
+     expect(order.add_item(menu_item, small_order)).to eq("#{small_order} #{menu_item.name}, added to your order")
```

Comment:
Metrics/LineLength: Line is too long. [114/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)